aspnetcore/signalr/javascript-client.md

```diff
@@ -0,0 +1,108 @@
+---
+title: ASP.NET Core SignalR JavaScript client
+author: rachelappel
+description: Create a JavaScript client for an ASP.NET Core SignalR hub.
+manager: wpickett
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 04/06/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/javascript-client
+---
+
+# ASP.NET Core SignalR JavaScript client
+
+By [Rachel Appel](http://twitter.com/rachelappel)
+
+The ASP.NET Core SignalR JavaScript client library enables developers to call server-side hub code.
+
+## Install the SignalR client package
+
+The SignalR JavaScript client library is delivered as an [npm](https://www.npmjs.com/) package. If you're using Visual Studio, call `npm install` from the command line in any folder. For Visual Studio Code, call the command from the **Integrated Terminal**.
```

Comment:
I think "**run** `npm install`" and "**run** the command from the Integrated Terminal" are clearer